src/interaction/interactionmode.cpp

```diff
@@ -0,0 +1,791 @@
+/*****************************************************************************************
```

Comment:
InteractionMode does not exist in master anymore. Any reason why this is added again?

Reply:
Good question. Removed.